builtin/providers/google/resource_sql_user_migrate.go

```diff
@@ -0,0 +1,39 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func resourceSqlUserMigrateState(
+	v int, is *terraform.InstanceState, meta interface{}) (*terraform.InstanceState, error) {
+	if is.Empty() {
+		log.Println("[DEBUG] Empty SqlUserState; nothing to migrate.")
```

Comment:
Actually InstanceState is a terraform type so you do want InstanceState here instead of SqlUserState